dvc/command/imp.py

```diff
@@ -82,4 +83,10 @@ def add_parser(subparsers, parent_parser):
             "This doesn't affect any DVC operations."
         ),
     )
+    import_parser.add_argument(
+        "--glob",
+        action="store_true",
+        default=False,
+        help="Allows importing paths containing shell-style wildcards.",
```

Comment:
```suggestion
        help="Allows import paths containing shell-style wildcards.",
```

Reply:
So it only affects the `path` argument right? Not `url`

Reply:
@jorgeorpinel  yes, that is my understanding.